# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: .cargo/config.in
# Commit: 1d5a44ae7207
# Full Hash: 1d5a44ae72071f079dcb00b62419c2ddd3c9f12c
# Author: Norisz Fay <nfay@mozilla.com>
# Date: 2023-01-27 21:11 +0200
# Regressor Bug: 1788119
# File Overlap Count: 1
# Overlapping Files: Cargo.lock
# Description:
#   Backed out 5 changesets (bug 1788119) for causing build bustages on DecoderData.cpp CLOSED TREE
# ==============================================================================

diff -r 83609f9575ac -r 1d5a44ae7207 .cargo/config.in
--- a/.cargo/config.in	Fri Jan 27 19:06:16 2023 +0000
+++ b/.cargo/config.in	Fri Jan 27 21:11:42 2023 +0200
@@ -102,7 +102,7 @@
 
 [source."https://github.com/mozilla/mp4parse-rust"]
 git = "https://github.com/mozilla/mp4parse-rust"
-rev = "eb0b625bd7e888d05ebcfc7685e2501b34c3b374"
+rev = "3bfc47d9a571d0842676043ba60716318e946c06"
 replace-with = "vendored-sources"
 
 [source."https://github.com/mozilla/neqo"]